substrate/frame/revive/src/wasm/runtime.rs

```diff
@@ -1790,17 +1790,20 @@ pub mod env {
 		&mut self,
 		memory: &mut M,
 		dest_ptr: u32,
+		dest_len: u32,
```

Comment:
It's just so annoying that we don't have a `FixedEncodedLen` trait. But we probably can get rid of SCALE for everything except XCM.

Reply:
Yeah I was thinking the same, this can probably be added to parity-scale-codec later on to make it a bit safer